src/Features/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.Features.csproj

```diff
@@ -26,6 +26,9 @@
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.CSharp" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.EditorFeatures.Wpf" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.Remote.Workspaces" />
+    <!-- BEGIN External Access -->
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.Razor" />
```

Comment:
Looks like we're the first ExternalAccess member to need this. I doing something wrong? 😄

Reply:
Some external access consumers have a dedicated assembly (F#, Razor). Others use Restricted IVTs.